CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [UNRELEASED] - YYYY-MM-DD
+
+### Changed
+
+- `collapse` and `graph` steps output parquet files
+- Update pixelator to 0.16.2
+
```

Comment:
Can you add the software versions table here like in the 1.0.1 release changelog?